newIDE/app/src/ExportAndShare/ExportPipeline.flow.js

```diff
@@ -32,19 +53,21 @@ export type ExportPipeline<
   CompressionOutput
 > = {|
   name: string,
-  onlineBuildType?: string,
+  onlineBuildType?: BuildType,
   limitedBuilds?: boolean,
   packageNameWarningType?: 'mobile' | 'desktop',
 
   getInitialExportState: (project: gdProject) => ExportState,
 
+  renderTutorial?: () => React.Node,
+
   renderHeader: (HeaderProps<ExportState>) => React.Node,
 
   shouldSuggestBumpingVersionNumber?: ({|
-    exportState: ExportState,
+    quota: ?Quota,
   |}) => boolean,
 
-  renderLaunchButtonLabel: () => React.Node,
+  renderExportFlow: (props: ExportFlowProps) => React.Node,
```

Comment:
To have better control about what to display for each export type and when to display it during the export, I've moved the launch button (or buttons) and the Stepper (if displayed) into this new function that encapsulate them.

Reply:
You're introducing this and removing renderCustomStepProgress so I'm ok with this 👍